callbacks/review_callbacks.py

```diff
@@ -6,18 +6,25 @@ def register_callbacks(app, spark_client: SparkClient):
   import dash
   import dash_bootstrap_components as dbc
   from components import intervals, grid, tabs, notification, actions
+  from utils import error_utils
 
   @dash.callback(
       dash.Input(intervals.initial_refresh_interval_id, "n_intervals"),
   )
   def refresh_cluster_initial(n_intervals):
-      spark_client.refresh_cluster("initial-" + str(n_intervals))
```

Comment:
i occassionally (pretty infrequently) see some strange cluster expired errors that are not caught by the spark_client. i'm not sure when they happen but if we don't catch here user will see some error notification. since this is meant to be an optimization just catching all exceptions here